### PR TITLE
(feat: plop-component-prettier):Added prettier command after a component is created

### PR DESCRIPTION
**Pull request by sowmihk**
Added prettier command after a component is created

### CODE REVIEW

File: templates/base/package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "storybook:install": "cd storybook && yarn",
     "storybook:run": "cd storybook && yarn storybook",
     "storybook:build": "cd storybook && yarn build-storybook",
-    "generate": "plop --plopfile generators/index.js",
+    "generate": "plop --plopfile generators/index.js && npx prettier --write src/components/",
```
Comment (vinodloha) [inline templates/base/package.json:17]: 1-2 issues here: 

1. NPX is going to install the prettier every-time where as we already have it as dependency. 
2. This command will be slow for whole project if they become really big, We want to run it on the component that is being generated only. 

So this you need to do it by using prettier API.

**Review by vinodloha — CHANGES_REQUESTED** [review]

**Review by vinodloha — CHANGES_REQUESTED** [review]
Please go through the comment and ask me if you have doubt.

File: templates/base/package.json (hunk 1 above)
Comment (sowmihk) [reply templates/base/package.json]: Changed to pretty-quick

File: templates/base/package.json [hunk 2]
```diff
@@ -14,10 +14,11 @@
     "storybook:install": "cd storybook && yarn",
     "storybook:run": "cd storybook && yarn storybook",
     "storybook:build": "cd storybook && yarn build-storybook",
-    "generate": "plop --plopfile generators/index.js",
+    "generate": "plop --plopfile generators/index.js && npm run pretty-quick",
     "bundlesize": "bundlesize --config ./config/bundlesize.json",
     "analyze": "cross-env ANALYZE=true next build",
-    "perf": "docker-compose -f ./config/performance/sitespeed-compose.yml run sitespeed.io  --config sitespeed.config.json --budget.configPath sitespeed.budget.config.json --graphite.host=graphite urls.txt"
+    "perf": "docker-compose -f ./config/performance/sitespeed-compose.yml run sitespeed.io  --config sitespeed.config.json --budget.configPath sitespeed.budget.config.json --graphite.host=graphite urls.txt",
+    "pretty-quick": "pretty-quick"
```
Comment (vinodloha) [inline templates/base/package.json:21]: I don't see pretty-quick or prettier installed in the dependencies.

Comment (sowmihk) [reply templates/base/package.json:21]: Added the dependencies

**Review by vinodloha — APPROVED** [review]

**Review by vinodloha — APPROVED** [review]

**Pull request merged**